.shippable.yml

```diff
@@ -152,6 +152,7 @@ build:
     - _make PLATFORM=rockchip-rk322x CFG_TEE_CORE_LOG_LEVEL=4 DEBUG=1
     - _make PLATFORM=sam
     - _make PLATFORM=marvell-armada7k8k CFG_ARM64_core=y
+    - _make PLATFORM=marvell-armada3700 CFG_ARM64_core=y CFG_ARM_GICV3=y
```

Comment:
`_make PLATFORM=marvell-armada3700` should be enough, that is, if you support only one configuration for now, make it the default (see below).

Reply:
Done